boot/bootdata/hivesft.inf

```diff
@@ -1776,7 +1777,7 @@ HKLM,"SOFTWARE\Microsoft\Ole","EnableRemoteConnect",0x00000000,"N"
 ; SvcHost services
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost",,0x00000012
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","DcomLaunch",0x00010000,"PlugPlay"
-HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","netsvcs",0x00010000,"DHCP","BITS","lanmanserver","lanmanworkstation","Schedule","Themes","winmgmt"
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\SvcHost","netsvcs",0x00010000,"DHCP","BITS","lanmanserver","lanmanworkstation","Schedule","Themes","winmgmt","W32Time"
```

Comment:
Note: W32Time is in the "LocalService" value instead (that one needs to be created).

Reply:
Yes, I agree. I was trying to keep these commits as simple as possible and it seemed that adding another Service was more than needed for now. Also, we need the netsvcs for this to work as all, so this seemed a good place to put this temporarily until we have the "LocalService". So maybe this could be fixed in a later PR?

Reply:
Do services in LocalService group have internet access? (for time syncing)

Reply:
They should (and they do, on Windows). Otherwise we can also put it in the NetworkService group.

Reply:
I tend to agree that this can wait until a future PR.

Reply:
No problem :)